consensus/src/quorum_store/utils.rs

```diff
@@ -501,20 +499,20 @@ impl ProofQueue {
             .observe(remaining_txns_without_duplicates as f64);
         //count the number of transactions with more than one batches
         counters::TXNS_WITH_DUPLICATE_BATCHES.set(
-            self.txn_summary_to_batches
+            self.txn_summary_num_occurrences
                 .iter()
-                .filter(|(_, batches)| batches.len() > 1)
+                .filter(|(_, count)| **count > 1)
```

Comment:
Is this necessary if `count == 0` are always removed on updates? Could it just be `self.txn_summary_num_occurences.count()`?

Reply:
`count == 0` are removed on updates. 
Here, I wanna see how many have `count > 1`  (ignoring `count == 1`)